wg-serving/charter.md

```diff
@@ -0,0 +1,90 @@
+# WG Serving Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [wg-governance].
+
+[Kubernetes Charter README]: /committee-steering/governance/README.md
+
+## Scope
+
+Discuss and enhance the support for AI/ML inference workloads in Kubernetes.
+
+By simplified definition, we will say that serving/inference workload is
```

Comment:
Serving and inference aren't interchangeable - thinking if there is a wording that could work better (this is not suggested wording, just trying to lay out the framing):

> Scope: Discuss and enhance serving workloads on Kubernetes, specifically focused on hardware-accelerated AI/ML inference

> Because serving and batch are both broad categories (pods with restartPolicy=Always vs those with restartPolicy=Never|OnFailure), we focus on the novel challenges of compute-intensive online inference, leveraging non-fungible accelerators vs highly generic CPU, and the unique reliability, efficiency, and cost obstacles those workloads face.  Our solutions should where possible provide automation that benefits other serving workloads like web services or stateful databases, be usable as primitives by multiple ecosystem projects, and compose well into the workflows of those deploying models to production.

How does that read?

Reply:
thank you! rephrased a little bit. WDYT?

Reply:
Looks good